### PR TITLE
Snappy ticket cleanup

### PR DESCRIPTION
**Pull request by gsvr**
Make a management command to remove all snappy tickets that do not have `nonce`s from the backend.

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: subscription/management/commands/snappy_ticket_cleanup.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    )
+
+    def handle(self, *args, **options):
+        tickets = Ticket.objects.filter(support_nonce=None)
+        counter = 0
+
+        self.stdout.write('Deleting tickets without support nonces...\n')
+        self.stdout.write('Nonceless tickets found: %s\n' % tickets.count())
+
+        for ticket in tickets:
```
Comment (gsvr) [inline subscription/management/commands/snappy_ticket_cleanup.py]: @imsickofmaps It's also possible in Django to delete all the objects by doing tickets.delete() - let me know if this is preferred - it is way quicker.

Comment (imsickofmaps) [inline subscription/management/commands/snappy_ticket_cleanup.py]: please change to your suggestion

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**